submarine-server/server-api/src/main/java/org/apache/submarine/server/api/Submitter.java

```diff
@@ -44,10 +44,11 @@
   /**
    * Create experiment with spec
    * @param spec experiment spec
+   * @param id experimentId 
    * @return object
    * @throws SubmarineRuntimeException running error
    */
-  Experiment createExperiment(ExperimentSpec spec) throws SubmarineRuntimeException;
+  Experiment createExperiment(ExperimentSpec spec, String experimentId) throws SubmarineRuntimeException;
```

Comment:
We already have `experimentId` in `ExperimentSpec`. 
Check `ExperimentSpec` -> `ExperimentMeta` -> `experimentId`

Reply:
oh i haven't notice before, will recover it.

Reply:
the code has been rollback.